capstone-overview/logistics.md

```diff
@@ -0,0 +1,12 @@
+## Capstone Logistics to Review
+
+1. Days off: The only holiday during capstone period is President's Day on February 15. Instructors will not be available to support on that day.
+1. Core hours: You can expect a staff member or volunteer TA to be holding office hours 9:30-12:00 and 1:00-3:00 at a minimum.
```

Comment:
Back when we were in person, students generally worked from home Tues/Thurs. I am wondering if this level of support is feasible for us as a time and necessary.